CSS Assignment 2/website.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+  <!--header starts-->
+  <head>
+    <title>
+      Information Page
+    </title>
+    <meta name = "author" content = "Akshita Dixit">
+    <link type="text/css" href="style.css" rel="stylesheet"/>
+    <link type="text/css" href="collapse.css" rel="stylesheet"/>
+    <link rel="stylesheet"

```

Comment:
Place custom CSS files after the third-party CSS files, so that styling information from your files overrides the third-party styles, instead of the other way around.